### ISSUE TITLE
Change order to 'asc' in createAlbum

### ISSUE THREAD

**Issue by ferbar** [Salvoxia/immich-folder-album-creator#36]
Hi,
thanks a lot for your work!

I want to suggest to create albums optionaly with order: 'asc'

at the moment it seems that this can only be achived by https://immich.app/docs/api/update-album-info

```
# Creates an album with the provided name and returns the ID of the
# created album
def createAlbum(albumName):
    apiEndpoint = 'albums'
    if version['major'] == 1 and version['minor'] <= 105:
        apiEndpoint = 'album'
    data = {
        'albumName': albumName,
        'description': albumName,
    }
    r = requests.post(root_url+apiEndpoint, json=data, **requests_kwargs)
    assert r.status_code in [200, 201]

    albumId = r.json()['id']

    if SOMEOPTION and version['major'] >= 1 and version['minor'] >= 99:
        data = {
            'order': 'asc',
        }
        r = requests.patch(root_url+apiEndpoint+f'/{albumId}', json=data, **requests_kwargs)
        assert r.status_code in [200, 201]
    return albumId

```

**Comment by Salvoxia** [comment]
Hm, I wasn't aware of that parameter.
Your use case is to have all albums created displaying oldest images first as opposed to the default?

**Comment by ferbar** [comment]
exactly!

What I want is the slide show in the right order. this works with order: 'asc'

**Comment by Salvoxia** [comment]
Thanks for clarifying!
Would you be interested in creating a PR request against the [dev branch](https://github.com/Salvoxia/immich-folder-album-creator/tree/dev) for that? You would be listed as a contributor then. (If so: Switches for different API versions have been removed in `dev` and 1.106.0 is the new minimum version required).
Otherwise I'll add it.

**Comment by ferbar** [comment]
created the PR, I hope this is fine

https://github.com/Salvoxia/immich-folder-album-creator/pull/37

**Comment by Salvoxia** [comment]
Released in [0.9.0](https://github.com/Salvoxia/immich-folder-album-creator/releases/tag/0.9.0).
